deface.gemspec

```
@@ -49,4 +50,9 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency('simplecov', '>= 0.6.4')
   spec.add_development_dependency('generator_spec', '~> 0.8')
   spec.add_development_dependency('pry')
+
+  if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('3.3')
```

RuboCop (1.75.5)

[warning] Do not use `RUBY_VERSION` in gemspec file.

(Gemspec/RubyVersionGlobalsUsage)